src/data/management.rs.in

```diff
@@ -0,0 +1,80 @@
+use time::Tm;
+
+use data::player::{Stats};
+use id::{Id,HasForgeableId};
+use data::{Map,Player,Monster};
+use entity::Entity;
+use instance::Instance;
+
+// Representation of an entity in the management API
+#[derive(Serialize,Debug,Clone)]
+pub struct EntityManagement {
+    pub id: Id<Entity>,
+    #[serde(rename="type")]
+    pub entity_type: EntityType,
+    pub skin: u64,
+    pub current_pv: u64,
+    pub position: PositionInstance,
+    pub stats: Stats,
+}
+
+#[derive(Serialize,Debug,Clone)]
+pub enum EntityType {
+    #[serde(rename="player")]
+    Player(PlayerStruct),
+    #[serde(rename="monster")]
+    Monster(MonsterStruct),
+}
+
+#[derive(Serialize,Debug,Clone)]
+pub struct PlayerStruct {
+    pub id: Id<Player>,
```

Comment:
I left the underlying representation of an `Id<T>` up to the type `T` via the trait `HasId`, so a certain type `T` can only have one single `Id` representation.

So effectively here the `Id<Player>` is effectively a `Uuid` underneath, whereas it is a `u64` for internal Lycan types.